lib/components/Navigation/NavItemDesktop.tsx

```diff
@@ -1,13 +1,41 @@
 import React, { FC } from "react";
 import * as styles from "./styles";
 
+export declare type LinkProps = {
```

Comment:
I don't think we need `export` or `declare` since its only used in this file, do we?
```suggestion
type LinkProps = {
```